config/components/manager/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
-resources:
-- manager.yaml
 # Comment this line if you want to disable the service for auth proxy (https://github.com/brancz/kube-rbac-proxy)
 # which protects your /metrics endpoint.
+resources:
```

Comment:
It's better to revert this back, because comment was addressed exact `auth_proxy_service.yaml`, not all resources

Reply:
In more that one target in the makefile  we use `kustomize edit` to set and reset some values (image),  if we don't change this, every time you run make you might end up "modifying" the files. 

This is especially problematic when extracting git tags, when the tag will get an extra "-dirty" mid run.

Reply:
comment updated.